examples/js/nuxtjs/nuxt.config.js

```diff
@@ -55,7 +55,20 @@ module.exports = {
     ** You can extend webpack config here
     */
     extend (config, ctx) {
+      if (!isDev && isClient) {
 
+        config.devtool = '#source-map'
```

Comment:
I think I saw somewhere in the logs that # was stripped because it was extraneous? https://webpack.js.org/configuration/devtool/#devtool

Reply:
Yes, you're right - I removed the #